hypershift-operator/controllers/hostedcluster/hostedcluster_controller.go

```diff
@@ -91,9 +91,10 @@ var NoopReconcile controllerutil.MutateFn = func() error { return nil }
 type HostedClusterReconciler struct {
 	client.Client
 
-	// HostedControlPlaneOperatorImage is the image used to deploy the hosted control
-	// plane operator.
-	HostedControlPlaneOperatorImage string
+	// HypershiftOperatorImage is the image used to deploy the control plane operator if
+	// 1) There is no hypershift.openshift.io/control-plane-operator-image annotation on the HostedCluster and
+	// 2) The OCP version being deployed is the latest version supported by Hypershift
+	HypershiftOperatorImage string
```

Comment:
Is this even needed now? The image comes from either a dynamic lookup or the annotation on the HC itself

Reply:
on `main` it still uses the hypershift operator image as the CPO image

Reply:
Gotcha, thanks!